analyzers/src/ITs.JsonParser/IssueParser.cs

```diff
@@ -58,11 +58,17 @@ private static IEnumerable<OutputReport> ExplodeToRuleIssues(InputReport inputRe
         var allIssues = inputReport.Sarif
             .AllIssues()
             .Where(x => RxSonarRule.IsMatch(x.RuleId))
-            .OrderBy(x => x.Order())
             .GroupBy(x => x.RuleId);
         foreach (var issuesByRule in allIssues)
         {
-            var issues = new RuleIssues { Issues = issuesByRule.Select(x => new RuleIssue(x)).ToArray() };
+            var issues = new RuleIssues
+            {
+                Issues = issuesByRule.Select(x => new RuleIssue(x))
+                    .OrderBy(x => x.Uri)
+                    .ThenBy(x => x.Location)
+                    .ThenBy(x => x.Message)
```

Comment:
Do this to fix the sorting issue
```suggestion
                    .OrderBy(x => x.Uri, StringComparer.InvariantCulture)
                    .ThenBy(x => x.Location, StringComparer.InvariantCulture)
                    .ThenBy(x => x.Message, StringComparer.InvariantCulture)
```